sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -445,6 +445,7 @@ object SQLConf {
         " this factor multiple the median partition size and also larger than " +
         s" ${ADAPTIVE_EXECUTION_SKEWED_PARTITION_SIZE_THRESHOLD.key}")
       .intConf
+      .checkValue(_ > 0, "The skew factor must be positive.")
```

Comment:
We no longer use `ADAPTIVE_EXECUTION_SKEWED_PARTITION_SIZE_THRESHOLD`, right? Let's remove it.

Reply:
When the `nonSkewSizes ` is very small. The `targetSize ` will be small without this config. Then it will split more small task when handling skewed partition? So we may need this config.

Reply:
We are using https://github.com/apache/spark/pull/27669/files#diff-2d6bea6eed43ca6f37fe3531cb574069R93 now, as we are trying to make the same target partition size for both coalesced non-skew partitions and skew partitions after split if the avg non-skew size is small.